src/shared/components/overlay/modal/Modal.tsx

```
@@ -10,15 +10,7 @@ const Modal = ({ isOpen, onClose, title }: ModalProps) => {
 
   return (
-    <dialog
-      className={styles.container}
-      role="dialog"
-      aria-modal="true"
-      // 모달 확인용 배경색 설정
-      style={{
-        border: '1px solid black',
-      }}
-    >
+    <dialog className={styles.container} role="dialog" aria-modal="true">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<dialog>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)